src/pages/questionPage.js

```diff
@@ -30,11 +31,10 @@ export const initQuestionPage = () => {
 
   document
     .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .addEventListener('click', nextQuestion);
+    .addEventListener('click', nextQuestionCheck);
 };
 
-const nextQuestion = () => {
+export const nextQuestion = () => {
```

Comment:
Don't export this function from this file. Because then `questionPage.js` will depend on `nextQuestionCheckUtil.js` AND `nextQuestionCheckUtil.js` will depend on `questionPage.js`. This is called `circular dependency` in software engineering and this is really bad practice. Try to send this function as an argument for nextQuestionCheck function